src/backend/user.js

```diff
@@ -0,0 +1,24 @@
+module.exports = class User {
```

Comment:
Zostałeś oszukany. `class` to pułapka ;)

https://medium.com/javascript-scene/how-to-fix-the-es6-class-keyword-2d42bb3f4caf#.13ga3iuia
https://medium.com/javascript-scene/the-two-pillars-of-javascript-ee6f3281e7f3#.gnz64l6xi

W skrócie - unikaj `new` `this` `class` wszędzie z wyjątkiem miejsc, gdzie wymaga tego znana i dobrze udokumentowana biblioteka. Nie organizuj kodu JSowego w stylu OOP

A w tym konkretnym przypadku - bardzo dziwne rzeczy trafiły do tej klasy...